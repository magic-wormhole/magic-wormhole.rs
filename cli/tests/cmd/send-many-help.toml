bin.name = "wormhole-rs"
args = "send-many --help"
