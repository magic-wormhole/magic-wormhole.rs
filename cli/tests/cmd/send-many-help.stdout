Send a file to many recipients

Usage: wormhole-rs[EXE] send-many [OPTIONS] <FILENAME|DIRNAME>...

Arguments:
  <FILENAME|DIRNAME>[..]

Options:
...

[..]sending the file in a loop[..]
...